src/main/java/ru/practicum/shareit/booking/Booking.java

```diff
@@ -12,26 +11,20 @@
 @Entity
```

Comment:
Data
Лучше не использовать эту аннотацию для сущностей. Дело в том, что ломбок генерирует EqualsAndHashCode, И могут быть конфликты с аналогичными механизмами hibernate’a

Reply:
Заменил на 
@Getter
@Setter

Reply:
Видимо мне требуется помощь.
Если я меняю аннотацию @data на 
@getter
@setter

у меня начинают падать интеграционные тесты, например
void saveUserTest() внутри UserServiceImplTest
ошибка 
org.mockito.exceptions.misusing.PotentialStubbingProblem: 
Strict stubbing argument mismatch.

почитал стэковерфлоу, можно добавить аннотацию
@MockitoSettings(strictness = Strictness.LENIENT)

если это сделать, интеграционный тест вообще перестает работать с ошибкой
java.lang.NullPointerException: Cannot invoke "ru.practicum.shareit.user.User.getId()" because "user" is null


В связи с этим, извиняюсь, но выставляю следующий PR без этих правок, но с учетом остальных замечаний.

Reply:
Как-то странно…. надо будет проверить, может быть поменяли тестовые скрипты.....